locale/en_US/locale.po

```diff
@@ -81,7 +82,7 @@ msgid "plugins.generic.pln.status"
 msgstr "Status"
 
 msgid "plugins.generic.pln.objectId"
-msgstr "Type Object Id"
+msgstr "Issue Id"
```

Comment:
@asmecher yes that is true. I removed the column that was referring to the type of the export, because, right now, the only export we are supporting is issue export, and because of that I changed the name of the related `ID` column. I could add both back, as they were. (or follow your suggestion and add another translation for `IssueId`)

Reply:
@asmecher I added a new locale key for IssueId.